src/topology/vector_bundle.lean

```diff
@@ -242,3 +266,192 @@ lemma is_topological_vector_bundle_is_topological_fiber_bundle :
+a bundle trivialization -/
+def local_triv_at (b : B) : topological_vector_bundle.trivialization R F Z.fiber :=
+Z.local_triv (Z.index_at b)
+
+lemma mem_source_at : (⟨b, a⟩ : total_space Z.fiber) ∈ (Z.local_triv_at b).source :=
+by { rw [local_triv_at, mem_local_triv_source], exact Z.mem_base_set_at b }
+
+@[simp, mfld_simps] lemma local_triv_at_apply : ((Z.local_triv_at b) ⟨b, a⟩) = ⟨b, a⟩ :=
+topological_fiber_bundle_core.local_triv_at_apply Z b a
+
+instance : topological_vector_bundle R F Z.fiber :=
+{ inducing := λ b, ⟨ begin refine le_antisymm _ (λ s h, _),
+  { rw ←continuous_iff_le_induced,
+    exact topological_fiber_bundle_core.continuous_sigma_mk ↑Z b, },
+  { refine is_open_induced_iff.mpr ⟨(Z.local_triv_at b).source ∩ (Z.local_triv_at b) ⁻¹'
+      (Z.local_triv_at b).base_set.prod s, (continuous_on_open_iff
+      (Z.local_triv_at b).open_source).mp (Z.local_triv_at b).continuous_to_fun _
+      (is_open.prod (Z.local_triv_at b).open_base_set h), _⟩,
+    rw [preimage_inter, ←preimage_comp, function.comp],
+    simp only [id.def],
+    refine ext_iff.mpr (λ a, ⟨λ ha, _, λ ha, ⟨Z.mem_base_set_at b, _⟩⟩),
+    { simp only [mem_prod, mem_preimage, mem_inter_eq, local_triv_at_apply] at ha,
+      exact ha.2.2, },
+    { simp only [mem_prod, mem_preimage, mem_inter_eq, local_triv_at_apply],
+      exact ⟨Z.mem_base_set_at b, ha⟩, }} end⟩,
+  locally_trivial := λ b, ⟨Z.local_triv_at b, Z.mem_base_set_at b⟩, }
+
+/-- The projection on the base of a topological vector bundle created from core is continuous -/
+lemma continuous_proj : continuous Z.proj :=
+topological_fiber_bundle_core.continuous_proj Z
+
+/-- The projection on the base of a topological vector bundle created from core is an open map -/
+lemma is_open_map_proj : is_open_map Z.proj :=
+topological_fiber_bundle_core.is_open_map_proj Z
+
+end topological_vector_bundle_core
+
+end
+
+variable [∀ x, topological_space (E x)]
+
+open topological_space
+
+/-- This structure permits to define a vector bundle when trivializations are given as local
+equivalences but there is not yet a topology on the total space. The total space is hence given a
+topology in such a way that there is a fiber bundle structure for which the local equivalences
+are also local homeomorphism and hence vector bundle trivializations. -/
+@[nolint has_inhabited_instance]
+structure topological_vector_prebundle :=
+(trivialization_at : B → topological_vector_prebundle.trivialization R F E)
+(mem_base_trivialization_at : ∀ x : B, x ∈ (trivialization_at x).base_set)
+(continuous_triv_change : ∀ x y : B, continuous_on ((trivialization_at x) ∘
+  (trivialization_at y).to_local_equiv.symm) ((trivialization_at y).target ∩
+  ((trivialization_at y).to_local_equiv.symm ⁻¹' (trivialization_at x).source)))
+(inducing : ∀ (b : B), @inducing _ _ _ (⨆ x : B,
+  coinduced (trivialization_at x).to_prebundle_trivialization.set_symm (subtype.topological_space))
+  (λ x : (E b), (id ⟨b, x⟩ : total_space E)))
```

Comment:
Can we do better here? Can we require something else which is easier to prove and from which we can prove this?